Tests/E2ETests/fss-e2e-tests.spec.ts

```diff
@@ -0,0 +1,123 @@
+        expect(await page.innerHTML(pageObjectsConfig.searchPageContainerHeaderSelector)).toEqual(pageObjectsConfig.searchPageContainerHeaderText);
+        
+        page.setDefaultTimeout(pageTimeOut.timeOutInMilliSeconds);
+        await SearchAttribute(page,"BusinessUnit");
+        await page.selectOption(pageObjectsConfig.operatorDropDownSelector,"eq"); 
+        await page.fill(pageObjectsConfig.inputSearchValueSelector,businessUnitValue);
+        await page.click(pageObjectsConfig.searchAttributeButton);
+        
+        // Verifiction of attribute table records
+        await page.waitForSelector(pageObjectsConfig.searchAttributeTable);
+        const noOfRecods=(await page.$$(pageObjectsConfig.searchAttributeTableRows)).length
+        expect(noOfRecods).toBeGreaterThanOrEqual(2);
+
+        //Get the token from local storage once user logged in
+        const idToken=await page.evaluate(()=>{return localStorage.getItem('idToken')})
+
+        //Search Query String
+        const queryString=`BusinessUnit eq '${businessUnitValue}'`
+        
+        //Validate api response status code matches 200 
+        await GetApiDetails(autoTestConfig.apiurl,queryString,idToken!,200);
+    
+      })
+
+      it('Valid search user attributes query to verify data returns on UI and API response status 200', async () => {
+
+        page.click(pageObjectsConfig.searchButtonSelector);
+        await LoginPortal(page,autoTestConfig.user, autoTestConfig.password);
+    
+        await page.waitForSelector(pageObjectsConfig.searchPageContainerHeaderSelector);
+        expect(await page.innerHTML(pageObjectsConfig.searchPageContainerHeaderSelector)).toEqual(pageObjectsConfig.searchPageContainerHeaderText);
+        
+        page.setDefaultTimeout(pageTimeOut.timeOutInMilliSeconds);
+        await SearchAttribute(page,"product");
+        await page.selectOption(pageObjectsConfig.operatorDropDownSelector,"eq"); 
+        await page.fill(pageObjectsConfig.inputSearchValueSelector,batchAttributeProduct);
+        await page.click(pageObjectsConfig.searchAttributeButton);
+
+        // Verifiction of attribute table records
+        await page.waitForSelector(pageObjectsConfig.searchAttributeTable);
+        const productNames = await page.$$eval(pageObjectsConfig.attributeTableDataSelector ,options => { return options.map(option => option.textContent) });
+    
+        for (let index = 0; index < productNames.length; index++) {
+        const productName = productNames[index];  
+        
+        expect(productName.toUpperCase()).toEqual(batchAttributeProduct.toUpperCase());            
+        }        
+        //Get the token from local storage once user logged in
+        const idToken=await page.evaluate(()=>{return localStorage.getItem('idToken')})
+
+        //Search Query String
+        const queryString=`$batch(product) eq '${batchAttributeProduct}'`
+        
+        //Validate api response status code matches 200 
```

Comment:
Should the code for the "for" be indented for readability (maybe it is just the GitHub display)

Reply:
I have done formatting for the same. Thanks